libs/utils/src/votes.ts

```diff
@@ -156,9 +156,7 @@ export function convertMarksToVotesDict(
         : /* c8 ignore next */
           throwIllegalValue(contest, 'type');
 
-    if (newVotes.length > 0) {
-      votesDict[mark.contestId] = [...existingVotes, ...newVotes] as Vote;
-    }
+    votesDict[mark.contestId] = [...existingVotes, ...newVotes] as Vote;
```

Comment:
Everything other than this is just testing updates.

Reply:
(within this commit)

Reply:
Do we need a regression test for the tally code where you first noticed this issue? I.e. a test that would have failed before that passes after this change?

Reply:
The tally code doesn't need a regression test, I don't think, because the tally code and report display code never changed. It was correctly handling the interpreter results. I don't think there was a good, clean way to address this issue at the tally code level.

I did want to make sure that the BMD and HMPB interpreters were outputting full `votes` dictionaries, and updating the existing tests covered that. I thought about adding am explicit test called "intepreter exports `votes` for all contests on page" but I think it's an extensive enough assertion now that the intent should be apparent.